docs/customization/usb-hardening.md

```diff
@@ -0,0 +1,153 @@
+
+# USB Hardening
+
+Qubes OS provides means to attach [USB devices to a qube](https://www.qubes-os.org/doc/how-to-use-usb-devices/)
+and in general recommends using an [USB qube](https://www.qubes-os.org/doc/usb-qubes/). The Qubes OS
+documentation also highlights the [security implications](https://www.qubes-os.org/doc/device-handling-security/).
+
+Nonetheless users may want to
+
+- harden their qubes with attached USB devices against USB attacks.
+- harden `dom0` against USB attacks, if they assigned some USB ports to `dom0`, e.g. for USB keyboard or mice
+  to avoid the [security pitfalls](https://www.qubes-os.org/doc/device-handling-security/#usb-security) of
+  assigning them to an [USB qube](https://www.qubes-os.org/doc/usb-qubes/).
+
+[USBGuard](https://usbguard.github.io/) is a tool that can be used for such purposes.
+
+## USBGuard
+
+### Security Considerations
+
+By default the Linux kernel will attempt to recognize plugged in USB devices based on their device ID
+and load the kernel drivers matching that device ID. If e.g. those device drivers have security issues,
+an attacker can exploit those and take control of the qube or in the case of `dom0` the entire system.
+
+[USBGuard](https://usbguard.github.io/) uses kernel interfaces to whitelist USB devices based on their
+provided interfaces and device IDs. Keep in mind though that rogue USB devices can easily forge device IDs
+and/or launch a [brute-force attack](https://en.wikipedia.org/wiki/Brute-force_attack) on an unknown
+whitelisted device ID.
+
+Rogue devices however cannot exploit interfaces/kernel drivers that are not made available to them due to
+[USBGuard](https://usbguard.github.io/). This is where hardening may make sense.
+
+### Installation
+
+[USBGuard](https://usbguard.github.io/) should be available from the default repositories of your
+[Qubes templates](https://www.qubes-os.org/doc/templates/).
+
+For dom0, you can install it via
```

Comment:
Could you please add some note around here mentioning risks of installing non-Qubes packages in dom0?